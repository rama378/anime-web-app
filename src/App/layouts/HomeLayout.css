.HomeLayout {
  --background: #000;
  --text-color: #fff;
  --font-family: 'Roboto', sans-serif;
  --font-size: 14px;
  --font-size-large: 16px;

  font-size: var(--font-size);
}