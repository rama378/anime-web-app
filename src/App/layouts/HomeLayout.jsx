import './HomeLayout.css';
import Footer from "../components/Footer/Footer";
import Header from "../components/Header/Header"

const HomeLayout = (props) => {
	return (
		<div className="HomeLayout">
			<Header />

			<div>
					{props.children}
			</div>

			<Footer />
		</div>
	);
};

export default HomeLayout;
