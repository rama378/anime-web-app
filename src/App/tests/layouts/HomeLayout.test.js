import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';

import HomeLayout from '../../layouts/HomeLayout';

const renderer = new ShallowRenderer();

describe('<HomeLayout />', () => {
  it('should render and match the snapshot', () => {
    renderer.render(<HomeLayout>Content</HomeLayout>);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
