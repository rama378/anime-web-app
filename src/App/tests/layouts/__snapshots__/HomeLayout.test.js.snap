// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeLayout /> should render and match the snapshot 1`] = `
<div
  className="HomeLayout"
>
  <Header />
  <div>
    Content
  </div>
  <Footer />
</div>
`;
